versions.json

```diff
@@ -35,16 +34,15 @@
       "buster/apache",
       "buster/fpm",
       "buster/zts",
+      "alpine3.16/cli",
+      "alpine3.16/fpm",
+      "alpine3.16/zts",
       "alpine3.15/cli",
       "alpine3.15/fpm",
-      "alpine3.15/zts",
-      "alpine3.14/cli",
-      "alpine3.14/fpm",
-      "alpine3.14/zts"
+      "alpine3.15/zts"
     ],
     "version": "8.0.19"
   },
-  "8.0-rc": null,
```

Comment:
Can we update this to keep these `null` entries? (these help @docker-library-bot make pre-releases automatically come and go :smile:)

I'm happy to make this change for you if you'd prefer :+1: